### DISCUSSION TITLE
What is `default scenario` in `script` argument?

### DISCUSSION THREAD

**Discussion by skasi-surya** [JetBrains/Qodana#137] [Q&A] [answered]
In the docker image configuration what does the `default` script do?

[Override the default run scenario](https://www.jetbrains.com/help/qodana/qodana-jvm-community-docker-techs.html#Override+qodana+workflow) states:

```
You can override the standard Qodana run scenario by using the --script option. By default, Qodana employs the default scenario, which is equivalent to running:

docker run ... jetbrains/qodana-jvm-community --script default
```

Will this script will be passed into  docker [RUN](https://docs.docker.com/engine/reference/builder/#run) command? How is it used in the context of Qodana for JVM ?

**Comment by brichbash** [comment, marked as answer]
At the moment, there are no scenarios for `qodana-jvm`. The default scenario means that Qodana will analyze your project as our IDEs would. You can see our ["PHP version migration" scenario](https://www.jetbrains.com/help/qodana/qodana-php-language-upgrade.html#cdaf828a) as an example of usage of `--script`.
